src/docs/PCW/PCWUserGuide.adoc

```diff
@@ -8,9 +8,9 @@ include::../all-doc-links.adoc[]
 == Main concepts of PCW
 
 *ProActive Cloud Watch (PCW)* monitors a system, and according to some rules, detects complex events, and triggers actions when configured events are fired. +
-PCW defines *rules*, which describe the metrics to monitor, triggering conditions and actions when the conditions are met. +
+PCW defines *rules*, which describe the metrics to monitor, triggering actions when conditions are met. +
```

Comment:
Proposal: PCW is configured with *rules* to defines the metrics to monitor, actions to performed, and conditions to be met to trigger them.

Reply:
@mcompastie it would be either "actions to _be_ performed" or "actions to _perform_", still, I think the current sentence is clearer.

Reply:
@luispinedamx do you prefer the original version of sentence in PR or this version:
"PCW is configured with rules to define the metrics to monitor, actions to be performed, and conditions to be met to trigger them."
?

Reply:
@ShatalovYaroslav I like your version better "PCW defines *rules*, which describe the metrics to monitor, triggering actions when conditions are met."  but I think it's a matter of taste, up to you

Reply:
Actually, I feel `triggering conditions and actions when the conditions are met.` could be made clearer, but it's just a proposal.

Reply:
I update according to your suggestions